crates/sui-framework/packages/dwallet-system/tests/native_dwallet_tests.move

```diff
@@ -23,8 +23,9 @@ module dwallet_system::native_dwallet_tests {
 
         let height: u64 = 10;
         let ctx = test_scenario::ctx(&mut scenario);
-	let 	(proof, prefix, path, value, root) = state_proof_test_data();
-        let client = init_lc(height, timestamp, next_validators_hash, root, ctx);
+	let (proof, prefix, path, value, root) = state_proof_test_data();
+        let client = init_client(height, vector[105, 98, 99, 45, 48], 0, 5 * 365 * 24 * 60 * 60, 40, ctx);
```

Comment:
can we use variables with names for the arguments? it will be easier to read the test

Reply:
Yes, I forgot do this on this file!